specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/examples/DaprComponents_CreateOrUpdate_SecretStoreComponent.json

```diff
@@ -0,0 +1,78 @@
+{
+  "parameters": {
+    "subscriptionId": "8efdecc5-919e-44eb-b179-915dca89ebf9",
+    "resourceGroupName": "examplerg",
+    "environmentName": "myenvironment",
+    "componentName": "reddog",
+    "api-version": "2022-06-01-preview",
+    "daprComponentEnvelope": {
+      "properties": {
+        "componentType": "state.azure.cosmosdb",
+        "version": "v1",
+        "ignoreErrors": false,
+        "initTimeout": "50s",
+        "secretsStoreComponent": "my-secret-store",
+        "metadata": [
+          {
+            "name": "url",
+            "value": "<COSMOS-URL>"
+          },
+          {
+            "name": "database",
+            "value": "itemsDB"
+          },
+          {
+            "name": "collection",
+            "value": "items"
+          },
+          {
+            "name": "masterkey",
+            "secretRef": "masterkey"
```

Comment:
>masterkey [](http://example.com/codeflow?start=26&length=9)

Should we have a new name format here? e.g. redis-secret/redis-password

Reply:
both `redis-secret` and `redis-secret/redis-password` are valid. The first will be translated to `redis-secret/redis-secret` in the RP.

I can make an example